arch/xtensa/src/esp32/esp32_wifi_adapter.c

```diff
@@ -638,26 +755,20 @@ static int esp_event_lock(bool lock)
 static void esp_set_isr(int32_t n, void *f, void *arg)
 {
   int ret;
+  uint32_t tmp;
   struct irq_adpt *adapter;
-  int irq;
-  int cpu = 0;
-  int tmp;
+  int irq = n + XTENSA_IRQ_FIRSTPERIPH;
 
-  if (g_wifi_irq >= 0)
-    {
-      return ;
-    }
+  wlinfo("INFO: n=%d f=%p arg=%p irq=%d\n", n, f, arg, irq);
```

Comment:
Could we improve the name of these variables? `n`, `f` ?
They are little expressive

Reply:
This is defined in esp-idf's https://github.com/espressif/esp-idf/blob/c49d03b8eb9cb2070960927cbd7194645c44a874/components/esp_wifi/include/esp_private/wifi_os_adapter.h#L38, I also add comment for this, I think we can keep it origin.

Reply:
ok